src/containers/Detail.js

```diff
@@ -0,0 +1,100 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView } from 'react-native';
+
+export const Detail = () => {
+  return (
+    <View style={styles.container}>
+      <ScrollView>
+      <View style={styles.info}>
+        <View style={styles.picture}></View>
+        <Text style={styles.name}>Name of house</Text>
+        <Text style={styles.location}>Location </Text>
+      </View>
+      <View style={styles.follow}>
+        <Text style={{color: 'white',paddingTop:5}}>Follow</Text>
+      </View>
+      <View style={styles.call}>
+        <Text style={{color: 'black', paddingTop:5}}>Call</Text>
+      </View>
+      <View style={styles.architecture}>
```

Comment:
- We will not use the picture like in the Figma prototype.
- Let just put some attributes of the house and the location of the house (can display by map).